linux_os/guide/services/usbguard/service_usbguard_enabled/rule.yml

```diff
@@ -0,0 +1,23 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Enable the USBGuard Service'
+
+description: |-
+    The USBGuard service should be enabled.
+    {{{ describe_service_enable(service="usbguard") }}}
+
+rationale: ""
```

Comment:
We typically explain why we suggest using the service. Check eg. rules service_chronyd_or_ntpd_enabled, service_rsyslog_enabled, service_firewalld_enabled for inspiration.

Reply:
Well, I've checked `service_sssd_enabled` and that is very silent about the rationale ...